### ISSUE TITLE
Save Cellpose-SAM results

### ISSUE THREAD

**Issue by AmberJar** [MouseLand/cellpose#1230]
# Prediction
model = models.CellposeModel(gpu=True,
                             pretrained_model='/scratch/pf2m24/projects/CellSeg/models/new_model')

masks, flows, _ = model.eval(test_data, z_axis=0, channel_axis=1,
                                batch_size=32,
                                do_3D=True, flow3D_smooth=1)

# Save
io.save_masks(test_data, masks, flows, test_list, png=False, tif=True, channels=[0, 0],
              suffix="_cp_masks", save_flows=False, save_outlines=False, dir_above=False,
              in_folders=False, savedir='/scratch/pf2m24/projects/CellSeg/cellpose_sam_results', save_txt=False, save_mpl=False)

# Question
While I use 'io.save_masks', nothing saved. How can I save my 3d results and have a look in the GUI?
Thank you!
